### DISCUSSION TITLE
Feature Request - Select All

### DISCUSSION THREAD

**Discussion by bankenichi** [BlafKing/sd-civitai-browser-plus#108] [Ideas]
Hello, it would be really great if there was an option to select all the 'Scan for:' options when you just want to do search for all available updates rather than having to check every single box.

**Comment by BlafKing** [comment]
Thanks for the request!

Sounds good, I'll include this into the next update 👍🏻

**Comment by BlafKing** [comment]
v2.0.1 now includes the "All" option when updating models! :)
